Source/Initialization/WarpXInitData.cpp

```diff
@@ -1240,7 +1240,8 @@ void WarpX::InitializeEBGridData (int lev)
               "particles are close to embedded boundaries");
         }
 
-        if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::Yee ||
+        if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::None ||
+            WarpX::maxwell_solver_id == MaxwellSolverAlgo::Yee ||
             WarpX::maxwell_solver_id == MaxwellSolverAlgo::CKC ||
             WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT) {
```

Comment:
There are several code blocks like this. Would it make sense to replace these with 
`if (WarpX::maxwell_solver_id != MaxwellSolverAlgo::PSATD)`
or do we prefer keeping it explicitly specifying the cases for which that block should be executed?

Reply:
Yes, I think it makes sense to have `!= MaxwellSolverAlgo::PSATD` here.

Reply:
Sounds good, thanks!